python/ray/util/placement_group.py

```diff
@@ -51,7 +51,7 @@ def bundle_reservation_check(placement_group):
         resource_name, value = self._get_none_zero_resource(bundle)
         num_cpus = 0
         num_gpus = 0
-        resources = None
+        resources = {}
```

Comment:
Random bug fix that is included in this PR. https://github.com/ray-project/ray/pull/11128